release/lts/debian11/test-deps/meta.json

```diff
@@ -11,6 +11,6 @@
         "test-deps-debian"
     ],
     "TestProperties": {
-        "size": 316
+        "size": 332
```

Comment:
size of what?

Reply:
The finalized image file

Reply:
It a test most teams like us do to make sure images don't grow unexpectedly